metagraph/src/graph/graph_extensions/node_rc.cpp

```diff
@@ -74,7 +75,10 @@ NodeRC::NodeRC(const DeBruijnGraph &graph, bool construct_index) : graph_(&graph
                     boss.bwd(*(it + 1)),
                     boss.get_node_last_value(*(it + 1)),
                     [&](edge_index incoming_boss_edge) {
-                        found |= (dbg_succ->boss_to_kmer_index(incoming_boss_edge) != DeBruijnGraph::npos);
+                        // TODO: make this more efficient?
+                        found |= (boss.get_minus_k_value(incoming_boss_edge,
```

Comment:
yeah, I guess it's better to precompute the mask and check with that as in traversal functions